### PR TITLE
Fix broken link in Service concept page

### PR DESCRIPTION
**Pull request by astraw99**
Fix the current `Ingress` 404 link, to make the doc better.

### CODE REVIEW

File: content/en/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -483,7 +483,7 @@ Kubernetes `ServiceTypes` allow you to specify what kind of Service you want.
 * `ClusterIP`: Exposes the Service on a cluster-internal IP. Choosing this value
   makes the Service only reachable from within the cluster. This is the
   default that is used if you don't explicitly specify a `type` for a Service.
-  You can expose the service to the public with an [Ingress](docs/reference/kubernetes-api/service-resources/ingress-v1/) or the
+  You can expose the service to the public with an [Ingress](/docs/concepts/services-networking/ingress/) or the
```
Comment (mengjiao-liu) [inline content/en/docs/concepts/services-networking/service.md:486]:
```suggestion
  You can expose the service to the public with an [Ingress](/docs/reference/kubernetes-api/service-resources/ingress-v1/) or the
```
I think the reason for 404 is that the `docs/reference/kubernetes-api/service-resources/ingress-v1/` address is missing `/`

Link https://kubernetes.io/docs/reference/kubernetes-api/service-resources/ingress-v1/  exists.

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm
/approve

File: content/en/docs/concepts/services-networking/service.md (hunk 1 above)
Comment (astraw99) [reply content/en/docs/concepts/services-networking/service.md:486]: Yes, but the old link is the ingress API fields doc, the new link `/docs/concepts/services-networking/ingress/` is better for ingress introduction.

Comment (mengjiao-liu) [reply content/en/docs/concepts/services-networking/service.md:486]: > Yes, but the old link is the ingress API fields doc, the new link `/docs/concepts/services-networking/ingress/` is better for ingress introduction.

Yes, I agree. The new link is better.

**Pull request merged**